script/pgsqlms

```diff
@@ -922,6 +929,14 @@ sub ocf_meta_data {
               <content type="integer" default="$pgport_default" />
             </parameter>
 
+           <parameter name="maxlag" unique="0" required="0">
+              <longdesc lang="en">
+	        Maximum admitted lag before we mark the server as inappropriate to promote
+              </longdesc>
+              <shortdesc lang="en">Max admitted lag</shortdesc>
+              <content type="integer" default="$maxlag_default" />
+            </parameter>
+
```

Comment:
"admitted" is probably not the best word here. 

I would suggest this as `shortdesc`:
```
Maximum lag before we mark a standby as inappropriate to promote
```

And this as `longdesc`:

```
Maximum lag allowed on a standby before we set a negative master score on it. 
This parameter must be a valid positive number as described in PostgreSQL documentation.
See: https://www.postgresql.org/docs/current/static/sql-syntax-lexical.html#SQL-SYNTAX-CONSTANTS-NUMERIC
```